libs/spark/src/index.ts

```diff
@@ -1,7 +1,18 @@
-export * from '@material-ui/core';
-// Have to be specific here, /core and /lab have overlapping exports.
-//  This is a fine reality though, /lab has few necessary component exports.
-export { Pagination, PaginationItem } from '@material-ui/lab';
+export {
+  Box,
+  CardMedia,
+  FormControl,
+  FormGroup,
```

Comment:
Preference: I think we should create respective files so all component re-exports follow the same pattern. (Box was a good call as a useful utility).

Reply:
I like this suggestion. It simplifies things if we decide later to wrap any of those components.